Recipes/Along-slope-velocities.ipynb

```diff
@@ -23,6 +23,7 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.    ax.set_title('Along-slope barotropic velocity (m s$^{-1}$)');
</pre><p>Stray <code>;</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/458/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Also, using a 7-CPU ARE instance, running this cell without pre-loading the alongslope_velocity data took about 90 mins</p>

Reply:
<p>This speed was back down to &lt;1 min once I put the time select back in.</p>